src/main/java/frc/robot/commands/PlaceCone/toZero.java

```diff
@@ -107,7 +105,7 @@ public void execute() {
     // System.out.printf(
     //     "X equals %.2f PID moves %.2f%n", poseEstimatorSubsystem.getAngle(), values[2]);
     // setpoint and atgoal don't work, just brute forced.
-    if (Math.abs(180 - poseEstimatorSubsystem.getAngle()) < 3) {
+    if (Math.abs((180 - poseEstimatorSubsystem.getAngle()) % 360) < 3) {
```

Comment:
Again, we need `MathUtil.inputModulus(180 - poseEstimatorSubsystem.getAngle(), -180, 180)`